spec/oauth2/access_token_spec.rb

```diff
@@ -75,6 +75,27 @@ def assert_initialized_token(target) # rubocop:disable Metrics/AbcSize
       assert_initialized_token(target)
       expect(target.expires_at).to be_a(Integer)
     end
+
+    describe 'expires_latency' do
+      it 'sets it via options' do
+        hash = {:access_token => token, :expires_latency => '10'}
+        target = described_class.from_hash(client, hash)
+        expect(target.expires_latency).to eq 10
+      end
+
+      it 'sets it 0 by default' do
+        hash = {:access_token => token, :expires_in => 100}
+        target = described_class.from_hash(client, hash)
+        expect(target.expires_latency).to eq 0
+      end
```

Comment:
Would prefer that it be `nil` by default.

Reply:
- [x] make it nil by default